java/code/src/com/redhat/rhn/frontend/action/rhnpackage/ExtraPackagesListAction.java

```diff
@@ -13,10 +13,6 @@
  * in this software or its documentation.
  */
 
-/**
- * Copyright (c) 2012 SUSE LLC
- */
```

Comment:
we should not remove our copyright. But we can add it now below the Red Hat one above. I think checkstyle allow this.